src/forms/forms.css

```diff
@@ -0,0 +1,114 @@
+.omrs-checkbox {
+  display: inline-block;
+  vertical-align: middle;
+  user-select: none;
+  cursor: pointer;
+}
+
+.omrs-checkbox input[type="checkbox"] {
+  opacity: 0;
+  position: absolute;
+}
+
+.omrs-checkbox label {
+  position: relative;
+  padding-left: 1.5rem;
+}
+
+.omrs-checkbox label::before,
+.omrs-checkbox label::after {
+  position: absolute;
```

Comment:
Why is this needed? What is its positioned ancestor?

Reply:
required this to position the `::after and ::before` content at the same position to implement the custom checkbox mark